Assets/Scripts/SS3D/Systems/Input/Controls.cs

```diff
@@ -0,0 +1,1025 @@
+//------------------------------------------------------------------------------
```

Comment:
I'm going to need some help understanding this one.

Reply:
This file is auto-generated class of Controls.inputactions, it changes every time you save changes in .inputacitons